presto-orc/src/main/java/com/facebook/presto/orc/OrcRecordReader.java

```diff
@@ -595,6 +614,49 @@ else if (type.getOrcTypeKind() == OrcTypeKind.MAP) {
         return statistics.build();
     }
 
+    /**
+     * @return All StreamReaders' local buffer sizes, plus all instance sizes
+     */
+    @VisibleForTesting
+    long getStreamReaderRetainedSizeInBytes()
+    {
+        long totalRetainedSizeInBytes = 0;
+        for (StreamReader column : streamReaders) {
+            if (column != null) {
+                totalRetainedSizeInBytes += column.getRetainedSizeInBytes();
+            }
+        }
+        return totalRetainedSizeInBytes;
+    }
+
+    /**
+     * @return StripeReader's buffer sizes, not the instance sizes
```

Comment:
`@return the size of memory retained by the current stripe (excludes object overheads)`